wagtail/core/models.py

```diff
@@ -1756,6 +1756,11 @@ def copy(self, recursive=False, to=None, update_attrs=None, copy_revisions=True,
         :param log_action flag for logging the action. Pass None to skip logging.
             Can be passed an action string. Defaults to 'wagtail.copy'
         """
+
+        # If the model doesn't exist in the DB, the operation will not work
+        if self._state.adding:
+            raise Exception('You cannot copy the unsaved object')
```

Comment:
Could we change this to a [`RuntimeError`](https://docs.python.org/3/library/exceptions.html#RuntimeError) instead?